sys/arm64/arm64/trap.c

```diff
@@ -427,15 +427,6 @@ data_abort(struct thread *td, struct trapframe *frame, uint64_t esr,
 		panic("data abort in critical section or under mutex");
 	}
 
-#if __has_feature(capabilities)
-	if ((esr & ISS_DATA_DFSC_MASK) == ISS_DATA_DFSC_LC_SC) {
-		sig = SIGSEGV;
-		ucode = (esr & ISS_DATA_WnR) == 0 ? SEGV_LOADTAG :
```

Comment:
Same comment about SEGV_STORETAG, but in this case, we probably also want to handle the SEGV_LOADTAG case here still as right now it just treats it as a plain VM_PROT_READ passed to vm_fault which is going to claim success and retry the instruction in an endless loop.